qemu/tests/cfg/numa_opts.cfg

```diff
@@ -92,3 +92,19 @@
               extra_params += " -numa node,mem=2560,cpus=5,nodeid=2"
               extra_params += " -numa node,mem=512,cpus=3-4,nodeid=0"
               extra_params += " -numa node,mem=1024,cpus=0-2,nodeid=1"
+    - nodes.128:
+        type = numa_maxnodes
+        host_node = 0
+        node_num = 128
```

Comment:
For other variants, `nodes.1`, `nodes.2`...   it use `numa_nodes = *` ,  but here it use `node_num = 128`.
Could you explain the difference between `numa_nodes` and `node_num`  ?

Reply:
Seems they have same meanning, have updated, thanks.